react/week3/src/Todo.js

```diff
@@ -0,0 +1,17 @@
+
+import React from "react";
+
+const Todo = (props) => {
+ 
+  return (
+    
+    <form id="myForm" onSubmit ={props.onSubmit} style={{ paddingLeft: 40, marginTop: 16 }}>
```

Comment:
Please try not to use inline styles in your react component

Reply:
Done.Thank you